src/components/sections/CommentFactory.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import Parse from 'parse/dist/parse.min.js';
+import '../../styles/index.css'
+import { List } from 'antd';
+import { Container } from 'react-bootstrap';
+
+
+function CommentFactory({ locationId }) {
```

Comment:
why is this a Factory? 🏭 :)

Reply:
Because it is a nice component that generates all the comments for each location :) We updated it to CommentSectionFactory